src/css/imports/dock.less

```diff
@@ -12,15 +9,21 @@
         cursor:pointer;
         float: left;
         position: relative;
+
+        // This allows it to be positioned beneath the logo
+        clear : right;
```

Comment:
IT can look odd with a larger logo and only one dock button.  If you float the buttons right an clear the dock left, then you can get the dock below the logo and right align the buttons.

![screen shot 2015-05-04 at 12 03 55 pm](https://cloud.githubusercontent.com/assets/9534192/7456918/5be509b8-f256-11e4-8cff-c5a563812b55.png)